### DISCUSSION TITLE
[2024-06-18] Incident Thread

### DISCUSSION THREAD

**Discussion by github-staff** [community/community#128981] [Enterprise] [answered]
We are investigating degraded performance for GitHub Enterprise Importer migrations.

_Subscribe to this Discussion for updates on this incident. Please upvote or emoji react instead of commenting +1 on the Discussion to avoid overwhelming the thread. Any account guidance specific to this incident will be shared in thread and on the [Incident Status Page](https://www.githubstatus.com/incidents/k1mlxlmdhqqp)._

**Comment by github-staff** [comment]
## Update

We are investigating degraded performance for GitHub Enterprise Importer migrations. Some customers may see an increase in failed migrations. Investigation is ongoing.

**Comment by github-staff** [comment]
## Update

We have identified what we believe to be the source of the migration errors and are applying a mitigation, which we expect will begin improving migration success rate.

**Comment by github-staff** [comment]
## Update

We have applied a configuration change to our migration service as a mitigation and are beginning to see recovery and in increase in successful migration runs. We are continuing to monitor.

**Comment by github-staff** [comment]
## Resolved

This incident has been resolved.

**Comment by github-staff** [comment, marked as answer]
Starting on June 18th from 4:59pm UTC to 6:06pm UTC, customer migrations were unavailable and failing. This impacted all in-progress migration during that time. This issue was due to an incorrect configuration on our Database cluster. We mitigated the issue by remediating the database configuration and are working with stakeholders to ensure safeguards are in place to prevent the issue going forward.

**Comment by amanaknows** [comment]
Run angelware protocol

**Comment by prakashca6789** [comment]
![IMG_20240723_070501_421](https://github.com/user-attachments/assets/3ab838ae-a440-4a9c-a46d-5f7f25670775)
